server/controllers/admin.py

```diff
@@ -637,9 +637,19 @@ def enrollment(cid):
     return render_template('staff/course/enrollment.html',
                            enrollments=students, staff=staff,
                            single_form=single_form,
+                           unenroll_form=forms.CSRFForm(),
                            courses=courses,
                            current_course=current_course)
 
+@admin.route("/course/<int:cid>/<string:user_id>/unenroll", methods=['POST'])
```

Comment:
Why is the `user_id` a string? 

Instead of having a specific endpoint for this route, it'd probably be better have that ID be part of the form data.